### DISCUSSION TITLE
I found a nice model on Hugging Face. Why is it not here on Civitai?

### DISCUSSION THREAD

**Discussion by Alyndiar** [civitai/civitai#224] [Q&A] [answered]
Like stated in the title, I've been using a nice model to make images. It's Elysium-Anime v3. I found it on Hugging Face and I'm wondering way it is not on Civitai. Can only the author upload it here? Can someone upload it instead? Or I could make and upload a remix with minimal influence. Since this model gives great results, I'd love to promote it to others :)

**Comment by JustMaier** [comment, marked as answer]
You can certainly upload it. If the author joins the site and wishes to take ownership of the model we have a process for verifying their ownership and transferring it to them 👍
